tasks_management/validation.py

```diff
@@ -39,19 +35,47 @@ def validate_create(cls, user, **data):
         super().validate_create(user, **data)
 
 
+class TaskValidation(BaseModelValidation):
+    OBJECT_TYPE = Task
+
+    @classmethod
+    def validate_update(cls, user, **data):
+        errors = validate_task(data)
+        if errors:
+            raise ValidationError(errors)
+        super().validate_create(user, **data)
+
+
 def validate_task_group(data, uuid=None):
     return [
         *validate_not_empty_field(data.get("code"), "code"),
         *validate_bf_unique_code(data.get('code'), uuid),
     ]
 
 
+def validate_task(data):
+    return [
+        *validate_task_status(data)
+    ]
+
+
 def validate_task_executor(data, uuid=None):
     return [
         *validate_user_exists(data.get("user_id"))
     ]
 
 
+def validate_task_status(data):
+    uuid = data.get('id', None)
+    instance = Task.objects.filter(id=uuid).first()
```

Comment:
It could be better to add .get(id=uuid) instead of filtering and returning first item in Queryset. Here below on line 71 you have reference to 'status'. What happen if instance is None due to having no object fulfilling such criteria?

Reply:
I changed that to get.